### PR TITLE
mv_compile - update

### PR DESCRIPTION
**Pull request by hansely**
-added a support for nnef & onnx models
-fixed a bug in nnir_to_clib.py to handle multiple outputs

### CODE REVIEW

**Comment by kiritigowda** [comment]
@hansely is this ready to be merged?

**Comment by hansely** [comment]
@kiritigowda You can merge after I update the names and @rrawther reviews the code. I'll let you know when it's ready.

File: model_compiler/python/nnir_to_clib.py [hunk 1]
```diff
@@ -952,43 +1000,44 @@ def generateModuleCPP(graph,fileName):
...
-                    printf("ERROR: mvAddToGraph: failed (%%d)\\n", status);
-                }
-                else if((status = vxVerifyGraph(handle->graph)) != VX_SUCCESS) {
-                    printf("ERROR: vxVerifyGraph: failed (%%d)\\n", status);
-                }
-                else {
-                    successful = true;
-                }
+            if((status = mvAddToGraph(handle->graph, %s, %s, binaryFilename)) != VX_SUCCESS) {
+                printf("ERROR: mvAddToGraph: failed (%%d)\\n", status);
```
Comment (rrawther) [inline model_compiler/python/nnir_to_clib.py:1017]: eventually we need to replace printf with some error code for the user

**Review by rrawther — APPROVED** [review]
@make sure the CI and tests are running

**Comment by kiritigowda** [comment]
@hansely need to merge this PR with #1108 , let me know if this is ready?

**Comment by hansely** [comment]
@kiritigowda you can merge the pr

**Pull request merged**